servicex_for_trexfitter/read_trex_config.py

```diff
@@ -0,0 +1,105 @@
+import re
+import linecache
+import json
+
+def read_configuration(confFile: str, block_name: str):
+    """
+    Read TRExFitter configuration file and return block as python dict
+    """
+    Block = {}
+    with open( confFile ) as configFile:
+        num = 0
+        for mark,line in enumerate(configFile.readlines()):        
+            if re.search(r'\b{}\b'.format(block_name), line):            
```

Comment:
In the long term: may prefer a proper parser... but as long as it works for now...

Reply:
Yeah, I will create an issue for this